src/app/components/paragraph/paragraph.component.html

```diff
@@ -0,0 +1,5 @@
+<div *ngIf="paragraph" class="paragraph">
```

Comment:
why not adding the CSS_PARAGRAPH[paragraph.typeContent] class on the first div (and make the div a section)

Reply:
would be nice indeed